databricks-catalog-external-location/catalogs.tf

```diff
@@ -0,0 +1,54 @@
+resource "databricks_catalog" "catalog" {
+  for_each       = { for idx, catalog in var.catalogs : catalog.name => catalog }
+  name           = each.value.name
+  storage_root   = "s3://${module.catalog_bucket.name}"
```

Comment:
I don't think you can/should have multiple catalogs pointing to the exact same storage root. Maybe add a prefix for the catalog name:
```
"s3://${module.catalog_bucket.name}/{each.value.name}"
```

Reply:
Hmm I'm pretty sure it is ok... the root is the same but then the location is different since each catalog gets put into a different folder within the root. I'll dm you some examples.

Reply:
Hmm, I still think an additional prefix would be a good idea, if for no other reason (assuming no uuid collision) at least making it easier to navigate